variables.sh

```diff
@@ -6,17 +6,18 @@ NAMESPACE=${NAMESPACE:-namely}
 CONTAINER=${DOCKER_REPO}${NAMESPACE}
 LATEST=${1:false}
 
-GO_VERSION=${GO_VERSION:-1.14}
-GRPC_VERSION=${GRPC_VERSION:-1.39}
-GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.39}
-GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.2.1}
-GRPC_GATEWAY_VERSION=${GRPC_GATEWAY_VERSION:-2.0.1}
-UBER_PROTOTOOL_VERSION=${UBER_PROTOTOOL_VERSION:-1.3.0}
-SCALA_PB_VERSION=${SCALA_PB_VERSION:-0.11.0}
+GO_VERSION=${GO_VERSION:-1.17}
+GRPC_VERSION=${GRPC_VERSION:-1.42}
+GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.42}
+GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.3.0}
+GRPC_GATEWAY_VERSION=${GRPC_GATEWAY_VERSION:-2.7.0}
```

Comment:
this ia major breaking change. I wouldn't mind upgrading at some point but I'd prefer not to do so in this release.
Is 2.0.1 not supported with go 1.17?

Reply:
> this ia major breaking change. I wouldn't mind upgrading at some point but I'd prefer not to do so in this release.
Is 2.0.1 not supported with go 1.17?

I don't recall there being any issues with Go 1.17 on the gRPC Gateway. Is this the `array`->`string` change I noticed? That does seem potentially breaking.

Happy to exclude that.

How about gRPC Web? Leave that be, or upgrade as shown...?

Reply:
yeah, the change from array to string would cause unexpected behavior, if it's not necessary I would dedicate a separate version for that upgrade.

I think in general I wouldn't upgrade anything that is not tied to go 1.17 or gRPC 1.42 (I actually expect the gRPC upgrade to also be released separately before we release the go upgrade, but it's fine to have right now because it's the same version that we'll end up using).

Reply:
> 

FYI, just confirmed that 2.0.1 included an update of its Go docker container to 1.15.3.